instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesExtractorTest.java

```diff
@@ -102,35 +147,65 @@ void shouldNotExtractTableIfAttributeIsNotSet() {
     underTest.onStart(attributes, context, request);
 
     // then
-    assertThat(attributes.build())
-        .containsOnly(
-            entry(DbIncubatingAttributes.DB_STATEMENT, "SELECT *"),
-            entry(DbIncubatingAttributes.DB_OPERATION, "SELECT"));
+    if (SemconvStability.emitOldDatabaseSemconv()) {
+      assertThat(attributes.build())
+          .containsOnly(
+              entry(DbIncubatingAttributes.DB_STATEMENT, "SELECT *"),
+              entry(DbIncubatingAttributes.DB_OPERATION, "SELECT"));
+    } else {
+      assertThat(attributes.build())
+          .containsOnly(
+              entry(
+                  SemconvStabilityUtil.getAttributeKey(DbIncubatingAttributes.DB_STATEMENT),
+                  "SELECT *"),
+              entry(
+                  SemconvStabilityUtil.getAttributeKey(DbIncubatingAttributes.DB_OPERATION),
+                  "SELECT"));
```

Comment:
I'd suggest changing these two lines to reference the stable names directly

Reply:
but we test both non-stable and stable. are you suggest to change gradle config to test stable only?

Reply:
this `else` condition is only executed when not emitting old db semconv

Reply:
yes, `SemconvStabilityUtil.getAttributeKey` checks for stability already.